main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math/rand"
+	"time"
+
+	"./createwindow"
+
+	"github.com/jezek/xgb"
+	"github.com/jezek/xgb/xproto"
+)
+
+func main() {
+	rand.Seed(time.Now().Unix())
+	ctxCh := make(chan context.Context)
+	X, wid, quitStruct, err := createwindow.Setup(ctxCh)
+	if err != nil {
+		errorHandler(err)
```

Comment:
Just inline log.Fatalln here for now? When we have an actual error handler, you can separate it (otherwise this bit looks weird in this cl)